### PR TITLE
Depts

### PR DESCRIPTION
**Pull request by cjwest**
@boznik 
This adds the new PICAL and department layouts.
please review.

### CODE REVIEW

File: modules/stanford_jumpstart_home_dept/stanford_jumpstart_home_dept.module [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ *  Implements hook_form_FORM_ID_alter()
+ *
+ * @param $form
+ * @param $form_state
+ */
+function stanford_jumpstart_home_dept_form_stanford_jumpstart_homepage_dashboard_form_alter(&$form, &$form_state) {
+
+  // Add the stanford_jumpstart_home_dept layouts to the customize design page
+  stanford_jumpstart_home_dashboard_add_layouts($form, $form_state, 'stanford_jumpstart_home_dept');
+  dpm($form);
```
Comment (sherakama) [inline modules/stanford_jumpstart_home_dept/stanford_jumpstart_home_dept.module]: Death to all dpm() functions in PRs.

Comment (jbickar) [reply modules/stanford_jumpstart_home_dept/stanford_jumpstart_home_dept.module]: 👍

Comment (jbickar) [reply modules/stanford_jumpstart_home_dept/stanford_jumpstart_home_dept.module]: Can we have codeclimate flag that as a fail?

Comment (sherakama) [reply modules/stanford_jumpstart_home_dept/stanford_jumpstart_home_dept.module]: Good question. This is on the SOE org so code climate is not running here. 
Maybe that is something we can add to Kellie's automated testing if CC does not allow for it.

**Comment by cjwest** [comment]
Okay @boznik, this is ready for your review!

**Pull request merged**